spec/requests/api/v1/invoice_items_find_all_spec.rb

```diff
@@ -0,0 +1,184 @@
+require 'rails_helper'
+
+RSpec.describe 'Invoice items API Find_All Controller' do
+  it 'can find all invoice item by id' do
+    test_invoice_items = create_list(:invoice_item, 3)
+    get "/api/v1/invoice_items/find_all?id=#{test_invoice_items.first['id']}"
+
+    invoice_items = JSON.parse(response.body)
+    invoice_item = invoice_items.first
+
+    expect(response).to be_success
+    expect(invoice_items.count).to eq(1)
+    expect(invoice_item).to be_a(Hash)
+    expect(invoice_item).to have_key('id')
+    expect(invoice_item['id']).to be_a(Integer)
+    expect(invoice_item).to have_key('item_id')
+    expect(invoice_item['item_id']).to be_a(Integer)
+    expect(invoice_item).to have_key('invoice_id')
+    expect(invoice_item['invoice_id']).to be_a(Integer)
+    expect(invoice_item).to have_key('quantity')
+    expect(invoice_item['quantity']).to be_a(Integer)
+    expect(invoice_item).to have_key('unit_price')
+    expect(invoice_item['unit_price']).to be_a(Integer)
+    expect(invoice_item).to have_key('created_at')
+    expect(invoice_item).to have_key('updated_at')
+  end
+
+  it 'can find all invoice item by item id' do
+    test_invoice_items = create_list(:invoice_item, 3)
+    test_invoice_items[0].update(item_id: 1)
+    test_invoice_items[1].update(item_id: 2)
+    test_invoice_items[2].update(item_id: 1)
+    get "/api/v1/invoice_items/find_all?item_id=#{test_invoice_items.first['item_id']}"
+
+    invoice_items = JSON.parse(response.body)
+    invoice_item = invoice_items.first
+
+    expect(response).to be_success
+    expect(invoice_items.count).to eq(2)
+    expect(invoice_item).to be_a(Hash)
+    expect(invoice_item).to have_key('id')
```

Comment:
@case-eee The thinking behind not comparing with actual JSON response, but checking each key and their class individually is that we had to go through several iterations of content formatting updates. In this case we can confirm the correct keys are coming through, but accurate content being returned we can make check at `model` level testing.

FYI @akintner